Private-Registry-for-Images.md

```diff
@@ -0,0 +1,37 @@
+## Using a Private Registry for Images
+
+When you create a function or deploy a service the image for the application/function will be pulled from the registry specified in the `--image` flag. In order to use a private registry for this image you need to provide a Secret containing your image pull credentials and add it to `imagePullSecrets` for the default service account in the namespace you are using for the application/function.
+
+When deploying images from a private Google Container Registry to the default namespace you could do the following:
```

Comment:
To make it a bit clearer that the directions below are for GCR only, I would start this paragraph with the words "For example, ..."